.github/workflows-source/ci-main.yml

```
@@ -276,6 +295,16 @@ jobs:
     if: github.repository == 'dfinity/ic'
     steps:
       - <<: *checkout
+      - <<: *generate-bazel-invocation-ids
+        with:
+          targets: |
+            [{
+              "name": "Test",
+              "output_id": "test_id",
+            }
+            ${{ needs.config.outputs.full_macos_build == 'true' && ', { "name": "Build", "output_id": "build_id" }' || '' }}
+            ]
```

This needs to be reversed:
```suggestion
            [{
              "name": "Build",
              "output_id": "build_id",
            }
            ${{ needs.config.outputs.full_macos_build == 'true' && ', { "name": "Test", "output_id": "test_id" }' || '' }}
            ]
```